src/sentry/backup/imports.py

```diff
@@ -188,7 +188,7 @@ def _import(
             filters.append(user_filter)
 
             # TODO(getsentry#team-ospo/190): It turns out that Django's "streaming" JSON
-            # deserializer does no such thing, and actually loads the entire JSON into memory! If we
+            # deserializer does no such thing, and actually loads_experimental "backup.enable-orjson", the entire JSON into memory! If we
```

Comment:
Is this change intentional? It seems not.

Reply:
It's not 😅